### DISCUSSION TITLE
Online maps

### DISCUSSION THREAD

**Discussion by PhieF** [organicmaps/organicmaps#446] [Ideas]
Hi,

One advantages of apps such as Gnome Maps or Magic Earth is that we can search for places and create routes without having to download maps... Even if we can if we want to

it would be great to be able to use online tiles and a search server for people not willing to download maps !

Thanks for your work

**Comment by mritzmann** [comment]
Would like to explain the motivation why I would find this great as well: In Switzerland, we have almost 100% mobile phone coverage. In addition, there is unlimited data volume for around CHF 25 ($27). Offline maps are not so widespread here (google maps, for example, does online by default, very few people change this. Of course, there are always exceptions).

**Reply by jimboolio** [reply]
Same thing in Finland

**Comment by kogelnik** [comment]
I love the offline maps for when I'm not well connected, like hiking in a remote spot.  I'm sure the app also regularly saved a lot battery by not needing a network transaction.  That's great.

Where online maps is very useful is for the occasional map access to look up something of interest.  If, for example, a user is not local to New York city and wants to look, because of the map tiling, there are at least 5 different downloads of about 50MB each just to browse at a reasonable detail.  

I've looked at a number of places and now the app is storing almost 1GB of data.   That data is then also included in app backups to my computer and the cloud. Having all that data on the phone and elsewhere and then needing to manage it, for what was a temporary look at a portion of it seems awfully wasteful.

**Reply by biodranik** [reply]
@kogelnik We disabled backups for downloaded maps in iOS, the next OM update will fix that issue.

**Comment by ghost** [comment]
What’s the progress?
